tests/mbed/enc/enc.c

```diff
@@ -182,9 +185,8 @@ int test(
     }
     else
     {
-        static const char* argv[] = {"test", "-v", "NULL"};
+        static const char* argv[] = {"test", "-v"};
```

Comment:
Can you explain the change to remove the test name?

Reply:
Credit to @EmilAlexandruStoica for this work. It's just clean up to remove a completely unused parameter; there used to be extra work to find and path the data file, but it didn't actually change the set of tests that were run. I'll update the commit message to reflect this.